### PR TITLE
Masonwoodford/statuses

### PR DESCRIPTION
**Pull request by masonwoodford**
I made some changes to the API while working on the ticket modal but decided to separate those out just to keep the final PR more focused and see if these changes were acceptable to the team.

### CODE REVIEW

File: src/api/types.ts [hunk 1]
```diff
@@ -36,15 +36,18 @@ export interface UserRecord {
...
 
 export interface StatusRecordOutput {
   id: number;
   object_uuid: string;
   title: string;
+  color: string;
   created: string;
   activated?: string;
   deactivated?: string;
+  due_date?: string;
```
Comment (tdimhcsleumas) [inline src/api/types.ts]: Not sure why a status should have a due_date.

File: src/api/types.ts [hunk 2]
```diff
@@ -90,7 +93,7 @@ export interface TicketFilterOptions {
 }
 
 export interface MemberRecord {
-  id: string;
+  id: number;
```
Comment (tdimhcsleumas) [inline src/api/types.ts]: This should be a string.

**Review by tdimhcsleumas — CHANGES_REQUESTED** [review]
The functions LGTM. Minor comments about types.

File: src/api/types.ts (hunk 1 above)
Comment (masonwoodford) [reply src/api/types.ts]: Whoops, should be for WriteTicketRecord.

File: src/api/statuses.ts [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    `/api/teams/${teamId}/statuses/${id}]`
+  );
+  return data;
+};
+
+export const listStatuses = async (
+  teamId: number,
+  page: number
+): Promise<PaginatedList<StatusRecordOutput>> => {
+  const { data } = await axiosInstance.get<PaginatedList<StatusRecordOutput>>(
```
Comment (tdimhcsleumas) [inline src/api/statuses.ts:45]: If this is intended to be paginated LGTM.

**Review by tdimhcsleumas — APPROVED** [review]

**Pull request merged**